src/Optical/ParaxialAnalysis/HeadEyeModel/Misc.jl

```diff
@@ -0,0 +1,76 @@
+
+
+# based on code from https://www.ilikebigbits.com/2015_03_04_plane_from_points.html
+"""
+    plane_from_points(points::Vector{SVector{3, Float64}}) ->  centroid, normal
+
+Estimate the best fitting plane for a set of points in 3D    
+"""
+function plane_from_points(points::Vector{SVector{3, Float64}}) 
+    if length(points) < 3 
+        return nothing      # At least three points required
+    end
+
+    sum = Base.sum(points)
+    centroid = Vec3(sum * (1.0 / (length(points))))
+
+    # Calc full 3x3 covariance matrix, excluding symmetries:
```

Comment:
there is a function Statistics.cov which I think computes the same thing. Would be simpler to use it, unless you are doing something non-standard. We already use the Statistics package so this wouldn't add a dependency.

Reply:
will do.